lib/install/management/delete.go

```diff
@@ -46,11 +47,36 @@ func (d *Dispatcher) DeleteVCH(conf *config.VirtualContainerHostConfigSpec) erro
 		return nil
 	}
 
-	if err = d.DeleteVCHInstances(vmm, conf); err != nil {
-		// if container delete failed, do not remove anything else
-		log.Infof("Specify --force to force delete")
+	d.parentResourcepool, err = d.getComputeResource(vmm, conf)
+	if err != nil {
+		log.Error(err)
+		if !d.force {
+			log.Infof("Specify --force to force delete")
+			return err
+		}
+		// Assume the cVMs and RP have already been deleted
+		log.Warnf("Proceeding with delete of VCH due to --force")
```

Comment:
Will there be later feedback / logging that no containerVMs have been removed?  If not should that be stated here?  i.e. "No container VMs found, but proceeding with delete of VCH due to use of --force"

IMO there's a possibility that the containerVMs exist else where, so they would be "orphaned". Just want to make sure the user understands that no other VMs will be removed...

Reply:
Good point. Added this